torch/_meta_registrations.py

```diff
@@ -260,6 +260,42 @@ def linalg_cholesky_ex(A: Tensor, upper: bool = False, check_errors: bool = Fals
     return L, infos
 
 
+@register_meta(aten.unsafe_split_with_sizes.default)
```

Comment:
can we reuse `split_with_sizes` decomposition and not add separate meta?

Reply:
Hmm do you know where I can find it? Seems like _meta_registrations.py doesn't have any reference to split_with_sizes.

Reply:
As far as I can see, you can also register this decomp https://github.com/pytorch/pytorch/blob/0998ec1e27b9d929275d43d324dd9342409f705c/torch/_decomp/decompositions.py#L1089 to `unsafe_split_with_sizes` and it should work

Reply:
ohh i see. Just updated to reflect this